app/controllers/v1/interventions/stars_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class V1::Interventions::StarsController < V1Controller
+  def make_starred
+    authorize! :read, Intervention
+
+    current_v1_user.stars.find_or_create_by(intervention_id: intervention_load.id)
+
+    render json: serialized_response(intervention_load, 'Intervention', params: { current_user_id: current_v1_user.id })
```

Comment:
I wonder if FE will use this response 🤔. Could you talk with @bartosz-kepka-htd and make sure? If he doesn't use maybe only status will be fine 🤷

Reply:
FE doesn't need that response